tools/pytorch2onnx.py

```diff
@@ -52,10 +53,31 @@ def pytorch2onnx(config_path,
         opset_version=opset_version)
 
     model.forward = orig_model.forward
+
+    # simplify onnx model
+    if do_simplify:
+        from packaging import version
```

Comment:
Seems to be new dependency. We may try not to use it by following [here](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/__init__.py).